package.json

```diff
@@ -28,6 +28,7 @@
     "debug": "^2.2.0",
     "inflection": "^1.7.1",
     "js-yaml": "^3.4.2",
+    "loopback-api-definition": "1.0.0",
```

Comment:
Please use `^1.0.0`, that way we don't have to change this value every time we publish a new backwards-compatible version of loopback-api-definition.